### PR TITLE
Feat: Landing page layout

### PR DESCRIPTION
**Pull request by bharati-21**
The landing page for the quiz app consists of the following-
- Header with hero text that displays the webapp name
- Category cards with images
- Category card description/ title along with CTA to start quiz
- The description and the CTA appears when user hovers over the card

Landing page image

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    justify-content: center;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: rgba(0,0,0,0.7);
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
```
Comment (Ronak-Mutha) [inline styles/styles.css]: You can probably add inset property here as shorthand for top, bottom, left, and right properties

**Review by Ronak-Mutha — COMMENTED** [review]
Code looks clean and good use of classes and properties. Good to merge.

**Review by samyakshah3008 — APPROVED** [review]
Things I liked about this PR: 

Clean code and UI 
Loved detailed description of preview for PR 
Good semantic html tags ✅ 
Good naming conventions 

Good for approval

**Comment by bharati-21** [comment]
> Code looks clean and good use of classes and properties. Good to merge.

Thank you for your review!

File: styles/styles.css (hunk 1 above)
Comment (bharati-21) [reply styles/styles.css]: Thanks for this suggestion. I have made the changes!

**Comment by bharati-21** [comment]
> Things I liked about this PR:
> 
> Clean code and UI Loved detailed description of preview for PR Good semantic html tags ✅ Good naming conventions
> 
> Good for approval

Thank you for your review!

**Pull request merged**